lib/bundler/dsl.rb

```diff
@@ -42,6 +42,7 @@ def eval_gemfile(gemfile, contents = nil)
 
     def gemspec(opts = nil)
       path              = opts && opts[:path] || '.'
+      glob              = opts && opts[:glob] #|| nil
```

Comment:
It makes it look symmetrical with the rest of the column, but is not really needed. I can remove it or uncomment it, whatever you want.